platform/microprofile/src/main/java/org/trellisldp/microprofile/TrellisServiceSupplier.java

```diff
@@ -32,19 +32,19 @@
  * triplestore resource service.
  */
 @ApplicationScoped
-class TrellisServiceSupplier {
+public class TrellisServiceSupplier {
 
     @Produces
-    private RDFConnection rdfConnection;
+    protected RDFConnection rdfConnection;
```

Comment:
Why `protected` here? Do we expect subclasses of this?

Reply:
Now that you ask, I don't really have a good rationale for that -- maybe `private` would be better.

Reply:
I'd go `private` unless/until there is a specific reason to do otherwise, but I'm old-fashioned like that.

Reply:
no problem, I'll change that back.